colossalai/inference/modeling/policy/nopadding_baichuan.py

```diff
@@ -42,17 +44,38 @@ def module_policy(self):
             ]
         )
 
+        # used for Baichuan 13B
+        policy["BaichuanLayer"] = ModulePolicyDescription(
+            sub_module_replacement=[
+                SubModuleReplacementDescription(
+                    suffix="mlp",
+                    target_module=NopadBaichuanMLP,
+                ),
+                SubModuleReplacementDescription(
+                    suffix="self_attn",
+                    target_module=NopadBaichuanAttention,
+                ),
+            ]
```

Comment:
Please extract the replacement description to reduce redundancy. For now the two layers use the same sub module replacement description.

Reply:
Done